bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/PullRequestInstance.java

```diff
@@ -67,11 +67,40 @@ String diffUrl() {
         return pr.getWebUrl() + ".diff";
     }
 
+    String changeUrl() {
+        return pr.getWebUrl() + "/files";
+    }
+
+    String changeUrl(Hash base, Hash head) {
+        return pr.getWebUrl() + "/files/" + base.abbreviate() + ".." + head.abbreviate();
```

Comment:
This is done differently on GitLab (not exactly sure what the mapping would be).